wagtail/admin/panels.py

```diff
@@ -221,7 +221,9 @@ def bind_to(self, model=None, instance=UNSET, request=UNSET, form=UNSET):
                 "%s.bind_to must be passed all of the arguments: instance, request, form"
                 % type(self).__name__
             )
+        return self.get_bound_panel(instance=instance, request=request, form=form)
 
+    def get_bound_panel(self, instance=None, request=None, form=None):
```

Comment:
Just wondering if we consider this private API, and if so should we underscore it?

Getting a bit confused by having two methods that are named almost the exact same: `bind_to` / `get_bound_panel`

Reply:
I see we've deprecated `bind_to` now!